### ISSUE TITLE
Reader: Post position & content blinking due to "is-placeholder" class being added/removed

### ISSUE THREAD

**Issue by mrfoxtalbot** [Automattic/wp-calypso#54866]
### Steps to reproduce the behavior

1. https://wordpress.com/read/feeds/12328231
2. Look at the way some posts in the feed are blinking

### Context
4168821-zen

#### Is this specific to the applied theme? Which one?
N/A

#### Does this happen on simple or atomic sites or both?
This is an AT Site.

#### Is there any console output or error text?
Nothing relevant, but I can see that the CSS class name "is-placeholder" would get added and removed from the "blinking" post.

#### Level of impact (Does it block purchases? Does it affect more than just one site?)
Only affecting this feed/site.

#### Reproducibility (Consistent, Intermittent) Leave empty for consistent.
I can reproduce this when using the Reader to visit this specific site both from my account and the user's account, but I have not seen it affecting other feeds.

#### Screenshot / Video:

![Screen Capture on 2021-07-25 at 09-12-35](https://user-images.githubusercontent.com/4452464/126890932-996372fe-ab81-41a2-958c-76e8897a3e75.gif)

**Comment by mrfoxtalbot** [comment]
For reference the current state of the Reader: p5PDj3-4Y8-p2

**Comment by carinapilar** [comment]
Reported on 32814652-hc too. I would like to point out that the workaround described here: p2EDhh-1k5-p2#comment-7466 - only works if run when the site is Public.

**Comment by retnonindya** [comment]
An A11N reported it here: p1655920366995129-slack-C02F2757J

Removed the duplicates and the flicker no more ✨

**Comment by druesome** [comment]
Another instance on https://oneoldcop.com/

Ticket: 7429808-zen

**Comment by retnonindya** [comment]
Huh. @druesome's ticket is... Interesting. Somehow, the feed includes comments too 🤔 I remove the feed items with "Comment on:..." and the feed is no longer flickering.

From what I can remember, comments should not get included on /feed. I'm not sure what happened, though. I checked the user's Reading and Discussion settings, and I found nothing. The site is AT, so perhaps some plugins caused the funkiness :/ The issue should be resolved now.
